CMakeLists.txt

```diff
@@ -1,13 +1,10 @@
 # Root CMake file for LStore
 cmake_minimum_required(VERSION 2.8)
 project(LStore C)
-if(NOT CMAKE_BUILD_TYPE)
-    set(CMAKE_BUILD_TYPE Debug)
-endif()
 
-if(NOT LSTORE_PROJECT_VERSION)
-    set(LSTORE_PROJECT_VERSION "0.5.1-unknown_commit")
-endif()
+set(CMAKE_BUILD_TYPE "Debug" CACHE STRING "Build type (Debug|Release)")
```

Comment:
Note that with the addition of -dbgsym package creation the meaning of CMAKE_BUILD_TYPE gets muddled. In this scenario you want your compile flags to be more like the debug flags and then the binaries later get separated into stripped release binaries and debug symbol files. This should be addressed somehow.